.github/workflows/Unit-Test.yml

```diff
@@ -19,4 +19,8 @@ jobs:
             CREW_REPO=${{ github.event.pull_request.head.repo.clone_url }} CREW_BRANCH=${{ github.head_ref }} crew update && \
             yes | crew upgrade && \
             yes | crew install vim && \
-            yes | crew remove vim"
+            yes | crew remove vim && \
+            git sparse-checkout add ../tests && \
```

Comment:
We can avoid this by just adding `tests` to `install.sh`. Could we do that?

We just have to change these lines:
```
  # Set sparse-checkout folders.
  git sparse-checkout set packages "manifest/${USER_SPACE_ARCH}" lib bin crew tools
  git reset --hard origin/"${BRANCH}"
```
to
```
  # Set sparse-checkout folders.
  git sparse-checkout set packages "manifest/${USER_SPACE_ARCH}" lib bin crew tests tools
  git reset --hard origin/"${BRANCH}"
```

Reply:
(Then the next image build will pick that up, and the image used for the workflows will have it.)

Reply:
Let me do a separate PR for that...